tests/backup/backup_helper.go

```diff
@@ -5140,37 +5140,43 @@ func checkBackupObjectForNonExpectedNS(ctx context.Context, backupName string) e
 	return nil
 }
 
-// getNamespaceAge gets the namespace age of all the namespaces on the cluster
-func getNamespaceAge() (map[string]time.Time, error) {
-	var namespaceAge = make(map[string]time.Time)
+// getPodAge gets the pod age of all pods on all the namespaces on the cluster
+func getPodAge() (map[string]time.Time, error) {
```

Comment:
I feel map[string][string]time.Time (map[namespace][pod]time.Time) would be more useful.

Reply:
Nice catch it would be much cleaner. It would slightly increase complexity but approach is much cleaner.